src/main/plugins/org.dita.html5/xsl/cover.xsl

```diff
@@ -127,8 +131,8 @@ See the accompanying LICENSE file for applicable license.
   <xsl:template match="*[contains(@class, ' map/map ')]" mode="chapterBody">
     <body>
       <xsl:apply-templates select="*[contains(@class, ' ditaot-d/ditaval-startprop ')]/@style" mode="add-ditaval-style"/>
-      <xsl:if test="@outputclass">
-        <xsl:attribute name="class" select="@outputclass"/>
+      <xsl:if test="(@outputclass ne '') or ($OUTPUTCLASS ne '')">
+        <xsl:attribute name="class" select="string-join(distinct-values((tokenize(@outputclass, '\s+'), tokenize($OUTPUTCLASS, '\s+'))), ' ')"/>
```

Comment:
If the values have leading whitespace, this will result in leading whitespace for the resulting `@class` attribute, but IMO it's ok to just leave it.